catalog/imaging/catalog-video-transformer/src/main/java/com/connexta/transformer/video/CopyPresentKlvProcessor.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Connexta, LLC
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformer.video;
+
+import java.util.Map;
+
+import ddf.catalog.data.Metacard;
+
+public class CopyPresentKlvProcessor implements KlvProcessor {
+
+    @Override
+    public void process(Map<String, KlvHandler> handlers, Metacard metacard) {
+        handlers.values()
+                .stream()
+                .distinct()
```

Comment:
In this case, I happen to know that the duplicate handlers being removed are always going to be the same actual object. So distinct is safe. However, implementing equals() will probably make it more clear.